prefixing.graphql

```diff
@@ -0,0 +1,54 @@
+schema
+  @core(using: "https://spec.example.com/specA/v1.0")
+  @core(using: "https://spec.example.com/specB/v2.0", as: "B") {
+  query: Query
+}
+
+"""
+specA_SomeType is a type defined by spec A.
```

Comment:
This file has a lot of single-underscored names that should be double-underscored.